Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/usr/local/lib/python3.8/site-packages/nbclient/client.py", line 1082, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/usr/local/lib/python3.8/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/usr/local/lib/python3.8/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/usr/local/Cellar/python@3.8/3.8.5/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.8/site-packages/nbclient/client.py", line 535, in async_execute
    await self.async_execute_cell(
  File "/usr/local/lib/python3.8/site-packages/nbclient/client.py", line 827, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/usr/local/lib/python3.8/site-packages/nbclient/client.py", line 735, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply['content'])
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
mordor_file = "https://raw.githubusercontent.com/OTRF/mordor/master/datasets/small/windows/"
registerMordorSQLTable(spark, mordor_file, "mordorTable")
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mUnboundLocalError[0m                         Traceback (most recent call last)
[0;32m<ipython-input-2-6ad82e227509>[0m in [0;36m<module>[0;34m[0m
[1;32m      1[0m [0mmordor_file[0m [0;34m=[0m [0;34m"https://raw.githubusercontent.com/OTRF/mordor/master/datasets/small/windows/"[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 2[0;31m [0mregisterMordorSQLTable[0m[0;34m([0m[0mspark[0m[0;34m,[0m [0mmordor_file[0m[0;34m,[0m [0;34m"mordorTable"[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m
[0;32m/usr/local/lib/python3.8/site-packages/openhunt/mordorutils.py[0m in [0;36mregisterMordorSQLTable[0;34m(spark, url, tableName)[0m
[1;32m     20[0m [0;34m[0m[0m
[1;32m     21[0m [0;32mdef[0m [0mregisterMordorSQLTable[0m[0;34m([0m[0mspark[0m[0;34m,[0m [0murl[0m[0;34m,[0m [0mtableName[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m---> 22[0;31m     [0mmordorJSONPath[0m [0;34m=[0m [0mdownloadMordorFile[0m[0;34m([0m[0murl[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m     23[0m     [0mmordorDF[0m [0;34m=[0m [0mreadJSON[0m[0;34m([0m[0mmordorJSONPath[0m[0;34m,[0m [0mspark[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m     24[0m     [0mmordorDF[0m [0;34m=[0m [0mprocessDataFrame[0m[0;34m([0m[0mmordorDF[0m[0;34m,[0m [0mspark[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;32m/usr/local/lib/python3.8/site-packages/openhunt/mordorutils.py[0m in [0;36mdownloadMordorFile[0;34m(url, dest)[0m
[1;32m     17[0m         [0mzipFile[0m [0;34m=[0m [0mZipFile[0m[0;34m([0m[0mBytesIO[0m[0;34m([0m[0mzipFileRequest[0m[0;34m.[0m[0mcontent[0m[0;34m)[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m     18[0m         [0mmordorJSONPath[0m [0;34m=[0m [0mzipFile[0m[0;34m.[0m[0mextract[0m[0;34m([0m[0mzipFile[0m[0;34m.[0m[0mnamelist[0m[0;34m([0m[0;34m)[0m[0;34m[[0m[0;36m0[0m[0;34m][0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m---> 19[0;31m     [0;32mreturn[0m [0mmordorJSONPath[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m     20[0m [0;34m[0m[0m
[1;32m     21[0m [0;32mdef[0m [0mregisterMordorSQLTable[0m[0;34m([0m[0mspark[0m[0;34m,[0m [0murl[0m[0;34m,[0m [0mtableName[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m

[0;31mUnboundLocalError[0m: local variable 'mordorJSONPath' referenced before assignment
UnboundLocalError: local variable 'mordorJSONPath' referenced before assignment

